R/plot_functions.R

```diff
@@ -158,17 +155,37 @@ plot_prob <- function(x, ...) {
 #' @rdname plot_prob
 #'
 #' @export
-plot_prob.cdf_estimation <- function(
-  x,
-  distribution = c(
-    "weibull", "lognormal", "loglogistic", "normal", "logistic", "sev"
-  ),
-  title_main = "Probability Plot",
-  title_x = "Characteristic",
-  title_y = "Unreliability",
-  title_trace = "Sample",
-  plot_method = c("plotly", "ggplot2"),
-  ...
+plot_prob.wt_model <- function(x,
```

Comment:
.wt_model... adjustment in NEWS ... or if you prefer weibulltools_* instead of wt_* then many changes have to be done :/